src/lib/errors.js

```diff
@@ -38,6 +38,10 @@ export function checkForCommonErrors() {
         $logger.warn(`ie_intranet_mode`);
     }
 
+    if (isIE() && !isIECompHeader()) {
+        $logger.warn(`Drop tag: <meta http-equiv="X-UA-Compatible" content="IE=edge">`);
```

Comment:
Yeah, the first string you pass to `$logger` should not have any spaces or special characters, because it gets converted to a CAL event on our server side, and used to build reports of event counts.